src/include/cart/api.h

```diff
@@ -423,6 +424,15 @@ crt_reply_get(crt_rpc_t *rpc)
 uint64_t
 crt_hlc_get(void);
 
+/**
+ * Return the second timestamp of hlc.
```

Comment:
need doxygen comment for hlc arg and return code

Reply:
not sure what is "doxygen comment", you mean the format of "@param"?
but all other API comment is with the format of "\param[in]"

Reply:
Yes, all other APIs doxygen is in the format of \param[in]  and \return, not sure why this one is in @-style